exercises/concept/party-robot/.docs/introduction.md

```diff
@@ -1,5 +1,9 @@
 # Introduction
 
+Go programs are organized into packages. 
+A package is a collection of source files in the same directory that are compiled together. 
+Functions, types, variables, and constants defined in one source file are visible to all other source files within the same package.
+
```

Comment:
This introduction should also include a short section on what is exported from a package.